gui/default/syncthing/device/editDeviceModalView.html

```diff
@@ -22,7 +22,7 @@
                 </ul>
               </p>
               <p class="help-block">
-                <span translate ng-if="deviceEditor.deviceID.$valid || deviceEditor.deviceID.$pristine">The device ID to enter here can be found in the "Actions > Show ID" dialog on the other device. Spaces and dashes are optional (ignored).</span>
+                <span translate ng-if="deviceEditor.deviceID.$valid || deviceEditor.deviceID.$pristine">The device ID to enter here can be found in the "Actions &gt; Show ID" dialog on the other device. Spaces and dashes are optional (ignored).</span>
```

Comment:
this will break the current translations as the original string changes

Reply:
We could possibly handle this in the tooling, translating back and forth as needed... (so that strings towards transifex get html entities resolved and removed, and re-added again when importing the translated strings.)

Reply:
The translation script is an ugly mess, but I think this should pretty much cover it:

```diff
--- a/script/translate.go
+++ b/script/translate.go
@@ -94,7 +94,7 @@ func inTranslate(n *html.Node, filename string) {
 }

 func translation(v string) {
-       v = strings.TrimSpace(v)
+       v = strings.TrimSpace(html.UnescapeString(v))
        if _, ok := trans[v]; !ok {
                av := strings.Replace(v, "{%", "{{", -1)
                av = strings.Replace(av, "%}", "}}", -1)
```

any HTML entities in the original source get parsed on the way to the translation JSON. I don't think we need to do anything in particular in the other direction (at least we haven't so far).

Reply:
Should I apply that change in this PR then?

Reply:
Yeah, try that, and run `go run build.go translate` on the changes afterwards (ideally it should not change the lang-en.json file).

Reply:
Actually, thinking about it one more time, it could be that the angular translate plugin fails to pick it up and we need to leave the html entities in the "key" string. So I guess see if translation still works, and if it doesn't it will need some adjustment.

Reply:
After `go run build.go translate`, my local copy of `lang-en.json` is empty.  With or without that change.

Reply:
I don't know why the `build.go` script fails, apparently the redirect of stdout does not work.  I always get an empty `lang-en-new.json` file.  But running `translate.go` does output correctly to stdout.

No change in content for `lang-en.json` with that patch applied.